packages/yourloops/pages/notifications/notification.tsx

```diff
@@ -294,19 +288,22 @@ export const Notification = (props: NotificationProps): JSX.Element => {
             }}
           />}
         {isAMonitoringInvitation && displayMonitoringTerms && notification.target &&
-          <MonitoringConsentDialog onAccept={acceptTerms} onCancel={() => { setDisplayMonitoringTerms(false) }}
+          <MonitoringConsentDialog onAccept={acceptTerms} onCancel={() => {
+            setDisplayMonitoringTerms(false)
+          }}
                                    teamName={notification.target.name} />
         }
         {props.userRole === UserRoles.caregiver && notification.type === NotificationType.careTeamProInvitation ? (
-          <IconButton
+          <Button
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏Here we have almost the same button defined twice in the ternary condition. The ternary condition could be removed, and a method `handleAcceptButtonClick()` should be created to handle the difference of behaviour between the case when a caregiver is invited to become a HCP, and the other cases